_distro_map.yml

```diff
@@ -156,7 +162,7 @@ openshift-dedicated:
     enterprise-3.11:
       name: '3'
       dir: dedicated/3
-    enterprise-4.13:
+    enterprise-4.14:
```

Comment:
@karanthshashank, please confirm that ROSA and OSD should start tracking 4.14 at OCP 4.14 GA.

Reply:
Yes ACK, we will merge to 4.14.